schema.sql

```diff
@@ -57,4 +57,36 @@ ALTER TABLE animals ADD COLUMN species_id INT references species(id);
 /* Add 'owner_id' column to animals table with foreign
    key referencing 'owners' table
 */
-ALTER TABLE animals ADD COLUMN owner_id INT references owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN owner_id INT references owners(id);
+
+/* Creating 'vets' table in 'vet_clinic' database
+   Table has columns: id, name, age, and date_of_graduation
+*/
+CREATE TABLE vets (
+    id  BIGSERIAL  NOT NULL,
+    name   TEXT NOT NULL,
+    age INT,
+    date_of_graduation DATE,
+    PRIMARY KEY(id)
+);
+
+/* Creating 'specialties' table in 'vet_clinic' database
+   Table has columns: id, specie, and vet
+*/
+CREATE TABLE specialties (
+    id  BIGSERIAL  NOT NULL,
+    vet TEXT,
+    specie TEXT,
+    PRIMARY KEY(id)
+);
+
+/* Creating 'visits' table in 'vet_clinic' database
+   Table has columns: id, animal, vet, and date_of_visited
+*/
+CREATE TABLE visits (
+    id  BIGSERIAL  NOT NULL,
+    animal TEXT,
+    vet TEXT,
```

Comment:
- [ ] Kinldy make sure to Remove id, animal and vet columns and add animal_id & vet_id columns inside the table which is of int data type.

Reply:
Hello @sja-thedude! Same here! The instruction on the project details does not specify what data type to use for the relation of the `animal` and `vet`.
![Screenshot from 2022-10-20 12-54-09](https://user-images.githubusercontent.com/101870993/196941578-55760990-be2b-41de-9759-1d04351846cd.png)

I choose to use text-based type with names (`animal` & `vet`) because it is easier to read than having numbers in form of `id`.

Please can you explain to me why it is necessary to use IDs instead of text?

Reply:
- [ ]  I have advised to use it as its a join table and ids are taken to access it. And ids take int as data type.

Reply:
I just removed the ID column for the `visits` table.
I initially used an ID to serve as a number log for registering all visits to the clinic.
Please help me share your opinion about using the ID in this table. But I have already taken it OFF.

Here is my table with the ID:
![Screenshot from 2022-10-20 13-22-19](https://user-images.githubusercontent.com/101870993/196949193-85131bfc-ce7c-44db-89a4-e8aee4bb89d5.png)

Here is the table without the ID:
![Screenshot from 2022-10-20 13-23-26](https://user-images.githubusercontent.com/101870993/196949207-7782ae09-0169-4c8c-b6be-f20bdbb2d0ff.png)